pubsub/cloud-client/subscriber.py

```diff
@@ -461,6 +618,46 @@ def callback(message):
     # [END pubsub_subscriber_error_listener]
 
 
+def receive_messages_with_delivery_attempts(
+    project_id, subscription_name, timeout=None
```

Comment:
Can we provide a default value for this timeout?

Reply:
It should default to None, and is expected to stream until the user enters `Ctrl+C` to terminate.